### PR TITLE
Update .gitignore

### PR DESCRIPTION
**Pull request by aymone**
Added common extensions to ignore.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
 /vendor
 /node_modules
+/bootstrap/compiled.php
+
 Homestead.yaml
 Homestead.json
+
+.DS_Store
+.env.*.php
+.env.php
```
Comment (crynobone) [inline .gitignore]: No longer in used in Laravel 5.x

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,19 @@
+#ignore:
+
 /vendor
 /node_modules
+/bootstrap/compiled.php
```
Comment (crynobone) [inline .gitignore]: No longer in used in Laravel 5.1.x+

File: .gitignore [hunk 3]
```diff
@@ -1,5 +1,19 @@
...
+
 Homestead.yaml
 Homestead.json
+
+.DS_Store
```
Comment (crynobone) [inline .gitignore]: Should be on your global gitignore

File: .gitignore [hunk 4]
```diff
@@ -1,5 +1,19 @@
...
+.env.*.php
+.env.php
 .env
+
+*~
+*.save
+*.bak
+*.old
+*.swp
```
Comment (crynobone) [inline .gitignore]: Should be on your global gitignore

**Comment by w2pc** [comment]
you should add these to your global .gitignore file. Not really check them in here.

**Pull request closed without merging**